cirq/experiments/fidelity_estimation.py

```diff
@@ -0,0 +1,70 @@
+"""Estimation of fidelity associated with experimental circuit executions."""
+
+from typing import Iterable, Tuple, Set
+
+import numpy as np
+
+from cirq.circuits import Circuit
+from cirq.ops import QubitOrder, QubitOrderOrList
+from cirq.sim import final_wavefunction
+
+
+def compute_linear_xeb_fidelity(
+        circuit: Circuit,
+        bitstrings: np.ndarray,
+        qubit_order: QubitOrderOrList = QubitOrder.DEFAULT,
+) -> float:
+    """Computes fidelity estimate from one circuit using linear XEB estimator.
+
+    Fidelity quantifies the similarity of two quantum states. Here, we estimate
+    the fidelity between the theoretically predicted output state of circuit and
+    the state producted in its experimental realization. Note that we don't know
+    the latter state. Nevertheless, we can estimate the fidelity between the two
+    states from the knowledge of the bitstrings observed in the experiment.
+
+    This estimation procedure makes two assumptions. First, it assumes that the
+    circuit is sufficiently scrambling that its output probabilities follow the
+    Porter-Thomas distribution. This assumption holds for typical instances of
+    random quantum circuits of sufficient depth. Second, it assumes that the
+    circuit uses enough qubits so that the Porter-Thomas distribution can be
+    approximated with the exponential distribution.
+
+    In practice the validity of these assumptions can be confirmed by plotting
+    a histogram of output probabilities and comparing it to the exponential
+    distribution.
+
+    In order to make the estimate more robust one should average the estimates
+    over many random circuits. The API supports per-circuit fidelity estimation
+    to enable users to examine the properties of estimate distribution over
+    many circuits.
+
+    See https://arxiv.org/abs/1608.00263 for more details.
+
+    Args:
+        circuit: Random quantum circuit which has been executed on quantum
+            processor under test
+        bitstrings: Results of terminal all-qubit measurements performed after
+            each circuit execution as integer array where each integer is
+            formed from measured qubit values according to qubit_order from
+            most to least significant qubit, i.e. in the order consistent with
+            cirq.final_wavefunction().
```

Comment:
Use back ticks when referring to a method like `cirq.final_wavefunction`. Omit the parens.